catalyst/utils/distributed.py

```diff
@@ -48,6 +48,15 @@ def check_apex_available() -> bool:
         return False and env_apex
 
 
+def check_amp_available() -> bool:
+    """Checks if torch.amp is available."""
+    major, minor = torch.__version__.split(".")[:2]
+    major = int(major)
+    minor = int(minor)
+    # torch.amp available since version 1.6 and above
+    return (major >= 1 and minor >= 6) or major > 1
```

Comment:
could you please use something like https://github.com/catalyst-team/catalyst/blob/9f97ad91c82322a18ac6a5764aaded7fc54720a4/catalyst/utils/seed.py#L27 ?

Reply:
Thanks, this way more cleaner way of checking version